docs/plugins/ingest-useragent.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[ingest-useragent]]
+=== Ingest Useragent Processor Plugin
+
+The Useragent processor extracts details from the useragent string a browser sends with its web requests. 
```

Comment:
The name of the HTTP header is `User-Agent`. So, it should be `userAgent` and `UserAgent`, that's the naming convention. Even the name of the plugin should be `ingest-user-agent`.